docs/tempo/website/setup/_index.md

```diff
@@ -1,40 +1,38 @@
 ---
-title: Set up a Tempo cluster
-menuTitle: Set up a Tempo cluster
-description: Learn how to set up a Tempo cluster and visualize data
+title: Set up a Tempo server or cluster
+menuTitle: Set up a Tempo server or cluster
+description: Learn how to set up a Tempo server or cluster and visualize data
 weight: 150
 ---
 
-# Set up a Tempo cluster
+# Set up a Tempo server or cluster
```

Comment:
I see you've added "server" in quite a few spots but it sounds weird to me. We don't like just "cluster"?

Reply:
The Tanka instructions set up a cluster while the linux instructions set up a single node. Cluster is no longer accurate for both instructions.

Reply:
Yeah. I agree. It's a little weird to me that these words "server" and "cluster" basically line up with "single binary" and "distributed" which we use elsewhere in the docs, but see what you're getting at.

FWIW there's nothing that prevents someone from running a Tempo cluster directly on a linux machine (as opposed to using some kind of orchestration), but I understand these instructions are focused on the single binary install.